### DISCUSSION TITLE
Welcome to MoodScapeBETA Discussions!

### DISCUSSION THREAD

**Discussion by molxeuz** [molxeuz/MoodScapeBETA#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by molxeuz** [comment]
First question #1

**Comment by gudodin** [comment]
First question #1 too

**Reply by molxeuz** [reply]
Yes

**Reply by molxeuz** [reply]
> First question #1 too

Yes again

**Reply by molxeuz** [reply]
> First question #1 too

Yes again
